mne/viz/_brain/_brain.py

```diff
@@ -234,22 +241,44 @@ def __init__(self, subject_id, hemi, surf, title=None,
                         y=self.geo[h].coords[:, 1],
                         z=self.geo[h].coords[:, 2],
                         triangles=self.geo[h].faces,
-                        color=self.geo[h].grey_curv,
+                        color=None,
+                        scalars=self.geo[h].bin_curv,
+                        vmin=geo_kwargs["vmin"],
+                        vmax=geo_kwargs["vmax"],
+                        colormap=geo_kwargs["colormap"],
+                        reverse_lut=geo_reverse,
                         opacity=alpha,
                     )
                     if isinstance(mesh_data, tuple):
-                        _, mesh = mesh_data
+                        actor, mesh = mesh_data
                         # add metadata to the mesh for picking
                         mesh._hemi = h
                     else:
-                        _, mesh = mesh_data, None
+                        actor, mesh = mesh_data, None
                     self._hemi_meshes[h] = mesh
+                    self._hemi_actors[h] = actor
                     self._renderer.set_camera(azimuth=views_dict[v].azim,
                                               elevation=views_dict[v].elev)
 
         if show:
             self._renderer.show()
 
+    def cortex_colormap(self, cortex, alpha):
+        """Return the colormap corresponding to the cortex."""
+        colormap_map = dict(classic=(dict(colormap="Greys",
+                                          vmin=-1, vmax=2,
+                                          opacity=alpha), False),
+                            high_contrast=(dict(colormap="Greys",
+                                                vmin=-.1, vmax=1.3,
+                                                opacity=alpha), False),
+                            low_contrast=(dict(colormap="Greys",
+                                               vmin=-5, vmax=5,
+                                               opacity=alpha), False),
+                            bone=(dict(colormap="bone",
+                                       vmin=-.2, vmax=2,
+                                       opacity=alpha), True))
```

Comment:
Instead of dealing with any of this `reverse_lut` business, I would just do `bone_r` as the colormap here. Then you don't need any to pass any `reverse_lut` code.